lti_consumer/lti_1p3/extensions/rest_framework/serializers.py

```diff
@@ -227,3 +230,90 @@ class Meta:
             'resultMaximum',
             'comment',
         )
+
+
+# pylint: disable=abstract-method
+class LtiContextSerializer(serializers.Serializer):
+    """
+    Serializer for a LTI Context
+    """
+    id = UsageKeyField()
+
+
+class LtiNrpsContextMemberBasicSerializer(serializers.Serializer):
+    """
+    Non PII fields serializer for Context Member.
+    """
+    status = serializers.SerializerMethodField()
+    user_id = serializers.SerializerMethodField()
+    roles = serializers.SerializerMethodField()
+
+    def get_user_id(self, user):
+        """
+        Get external user id for the user.
+        """
+        externalid, _ = compat.get_or_create_externalid(user)
+        return serializers.UUIDField().to_representation(externalid.external_user_id)
+
+    def get_status(self, user):
```

Comment:
The `status` field in the NRPS service means if the membership is active. Currently, we only fetch users that have active enrollments in the course, so the answer here should always be `True`.
We should not return inactive users through this endpoint.